inspire_utils/name.py

```diff
@@ -76,11 +77,15 @@ def __init__(self, name, constants=None):
             constants = ParsedName.constants
 
         self._parsed_name = HumanName(name, constants=constants)
+        self._parsed_name.capitalize()
 
         # In the case of one name part, i.e. only lastname, name parser adds the name part in one of the first, middle,
         # suffix fields (depending on input). For our use-case if a user types a name, usually would be a lastname.
-        if len(name.split()) == 1:
-            self._parsed_name.last = self._parsed_name.first + self._parsed_name.middle + self._parsed_name.suffix
+        if len(re.split('[^\w]', name)) == 1:
```

Comment:
This fails, even when being unicode-aware (`re.UNICODE` flag):
```
In [4]: re.split('[^\w]', u'héllo', re.UNICODE)
Out[4]: [u'h', u'llo']
```

Reply:
@michamos I've updated the logic of checking whether multiple name parts exist in the input and moved it [here](https://github.com/inspirehep/inspire-utils/pull/29/files#diff-f116d82c04785f5d37d57c86cae932cdR94).